.github/workflows/ci.yml

```diff
@@ -60,6 +60,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # fetch all tags and branches
```

Comment:
Idk if there is a way that we can only fetch `origin/main` for this action

Reply:
What's the problem you're trying to solve here?

Reply:
Oh, when running pylint we only want to run against changed files, so we need to fetch origin main

I think that we should just run git fetch origin main instead of `fetch-depth: 0`?

Reply:
Yeah, that would be better.